src/billing/utils.ts

```diff
@@ -21,3 +23,31 @@ export const isCreating = (project: BillingProject) =>
   project.status === 'Creating' || project.status === 'CreatingLandingZone';
 export const isDeleting = (project: BillingProject) => project.status === 'Deleting';
 export const isErrored = (project: BillingProject) => project.status === 'Error' || project.status === 'DeletionFailed';
+
+export const billingAccountIconSize = 16;
+
+export type BillingAccountStatus = 'updating' | 'done' | 'error';
+const billingAccountIconProps: Record<BillingAccountStatus, IconProps> = {
+  updating: {
+    icon: 'sync',
+    color: colors.warning(),
+    size: billingAccountIconSize,
+    'aria-label': 'billing account updating',
```

Comment:
These icons should have labels because they are the only indication of the billing account status (when displayed)… they are not simply decorative.

Example of them rendered (code hacked, as normally they don't display if all billing accounts are up-to-date):
<img width="819" alt="image" src="https://github.com/user-attachments/assets/6240f394-f4e3-4136-859f-a8183ce844a3">